viewport/store/selectors.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Returns true if the viewport matches the given query, or false otherwise.
+ *
+ * @param {Object} state Viewport state object.
+ * @param {string} query Query string. Includes operator and breakpoint name,
+ *                       space separated. Operator defaults to >=.
+ *
+ * @example
+ *
+ * ```js
+ * isViewportMatch( state, '< huge' );
+ * isViewPortMatch( state, 'medium' );
+ * ```
+ *
+ * @return {boolean} Whether viewport matches query.
+ */
+export function isViewportMatch( state, query ) {
+	const key = [ '>=', ...query.split( ' ' ) ].slice( -2 ).join( ' ' );
```

Comment:
Minor: I had to spend some time to understand what's happening here. It might be only me, so feel free to ignore. It might be more code to express it differently, but maybe it would pay off. Example:

```js
const key = query.split( ' ' ).length === 1 ? `>= ${ query }` : query;
```

I think it will work with the existing tests.

Reply:
>Minor: I had to spend some time to understand what's happening here.

Yeah, it's one of my favorite "clever" codes that I still find myself leaning on. 😄 I might argue it's a valuable pattern that we can introduce developers to. For example, I've also found value in using it to format a date as MM-DD-YYYY (where MM and DD can be one or two digits):

```js
[ d.getMonth() + 1, d.getDate() ].map( ( n ) => ( '0' + n ).slice( -2 ) ).concat( d.getFullYear() ).join( '-' )
```

Maybe some clarifying comments explaining the behavior?

Reply:
```js
const lastTwo = items => items.slice( - 2 );
cons key = lastTwo( [ '>=', ...query.split( ' ' ) ] ).join( ' ' );
```
For me personally, this would help :)

Reply:
>For me personally, this would help :)

https://lodash.com/docs/4.17.5#takeRight 😄

Reply:
Added an inline comment and used Lodash's `_.takeRight` in rebased 217021a.